src/lib/core/theming/_theming.scss

```diff
@@ -14,11 +14,12 @@
 // @param $color-map
 // @param $primary
 // @param $lighter
-@function mat-palette($base-palette, $default: 500, $lighter: 100, $darker: 700) {
+@function mat-palette($base-palette, $default: 500, $lighter: 100, $darker: 700, $text: 500) {
```

Comment:
`text` -> `text-hue`?

Reply:
I kept it to `$text` for consistency with `$default` etc - but happy to change it

Reply:
That's fair- they're all hues now that I think about it.